bundles/pom.xml

```diff
@@ -22,6 +22,8 @@
     <module>org.openhab.core.automation.module.media</module>
     <module>org.openhab.core.automation.module.script</module>
     <module>org.openhab.core.automation.module.script.rulesupport</module>
+    <module>org.openhab.core.automation.module.script.scriptenginefactory.jython</module>
+    <module>org.openhab.core.automation.module.script.scriptenginefactory.nashorn</module>
```

Comment:
Should this be included in this PR? Or the separate nashorn one?

Reply:
Thank you... yes, this is for another PR and will be removed.